btcjson/btcwalletextcmds.go

```diff
@@ -93,6 +93,84 @@ func NewRenameAccountCmd(oldAccount, newAccount string) *RenameAccountCmd {
 	}
 }
 
+// PsbtInput represents an input to include in the PSBT created by the
+// WalletCreateFundedPsbtCmd command.
+type PsbtInput struct {
+	Txid     string `json:"txid"`
+	Vout     int64  `json:"vout"`
+	Sequence int64  `json:"sequence"`
+}
+
+// PsbtOutput represents an output to include in the PSBT created by the
+// WalletCreateFundedPsbtCmd command.
+type PsbtOutput map[string]float64
+
+// NewPsbtOutput returns a new instance of a PSBT output to use with the
+// WalletCreateFundedPsbtCmd command.
+func NewPsbtOutput(address string, amount float64) PsbtOutput {
+	out := make(map[string]float64)
+	out[address] = amount
+	return out
+}
+
+// WalletCreateFundedPsbtOpts represents the optional options struct provided
+// with a WalletCreateFundedPsbtCmd command.
+type WalletCreateFundedPsbtOpts struct {
+	ChangeAddress          *string  `json:"changeAddress,omitempty"`
+	ChangePosition         *int64   `json:"changePosition,omitempty"`
+	ChangeType             *string  `json:"change_type,omitempty"`
```

Comment:
Should be an enum with values `legacy`, `p2sh-segwit`, and `bech32`. Let's try to update the `FundRawTransactionOpts` struct for this if possible.